spec/requests/api/v1/payments_batches_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
```

Comment:
I assume this is no-where near enough tests, but I'm also not too sure what we want to test here?
Maybe @katelovescode can give a couple test cases :)))))